<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>1101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Predict</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="PlotWidget_trajectory" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nu_1_0">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>nu_1(0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>nu_2(0)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nu_2_0">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>110</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>150</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>nu_3(0)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nu_3_0">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>180</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>nu_1(T)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nu_1_T">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>nu_2(T)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nu_2_T">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>110</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nu_3_T">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>180</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>150</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>nu_3(T)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x(T)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x_T">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y(T)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y_T">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>110</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_theta_T">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>180</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>150</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>theta(T)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>240</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>T</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_t_sw">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>270</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>240</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>t_sw</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_T">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>270</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_compute">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>270</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_nu_3" native="true">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>350</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_nu_1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_nu_2" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_u_1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_u_2" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>690</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_u_3" native="true">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>690</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_theta" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>150</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>theta(0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x(0)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x_0">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y(0)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y_0">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>110</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_theta_0">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>180</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1335</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
